lib/konbata/configuration.rb

```diff
@@ -2,13 +2,23 @@
 
 module Konbata
   class Configuration
+    attr_accessor :canvas_url
+    attr_accessor :canvas_token
+    attr_accessor :account_id
+    attr_accessor :request_timeout
```

Comment:
That was just the way that senkyoshi had them set up but I think that sounds like a better way to do it.